src/main/kotlin/com/cjbooms/fabrikt/generators/model/QuarkusReflectionModelGenerator.kt

```diff
@@ -0,0 +1,28 @@
+package com.cjbooms.fabrikt.generators.model
+
+import com.cjbooms.fabrikt.cli.ModelCodeGenOptionType
+import com.cjbooms.fabrikt.model.Models
+import com.cjbooms.fabrikt.model.QuarkusReflectionModel
+import com.cjbooms.fabrikt.model.ResourceFile
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+
+class QuarkusReflectionModelGenerator(
+    private val models: Models,
+    private val options: Set<ModelCodeGenOptionType> = emptySet()
+) {
+    companion object {
+        const val RESOURCE_FILE_NAME = "reflection-config.json"
```

Comment:
The problem with having a hard-coded file name in `/src/main/resources` is that multiple generate tasks will overwrite each other